SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -71,13 +72,15 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 	 * GetMinimumRequiredWorkers and SetVirtualWorkerIds are used to assign ranges of virtual worker IDs which will be managed by this
 	 * strategy. LastVirtualWorkerId - FirstVirtualWorkerId + 1  is guaranteed to be >= GetMinimumRequiredWorkers.
 	 */
-	virtual uint32 GetMinimumRequiredWorkers() const
-		PURE_VIRTUAL(UAbstractLBStrategy::GetMinimumRequiredWorkers,
-					 return 0;) virtual void SetVirtualWorkerIds(const VirtualWorkerId& FirstVirtualWorkerId,
-																 const VirtualWorkerId& LastVirtualWorkerId)
+	virtual uint32 GetMinimumRequiredWorkers() const PURE_VIRTUAL(UAbstractLBStrategy::GetMinimumRequiredWorkers, return 0;)
+
+		virtual void SetVirtualWorkerIds(const VirtualWorkerId& FirstVirtualWorkerId, const VirtualWorkerId& LastVirtualWorkerId)
 			PURE_VIRTUAL(UAbstractLBStrategy::SetVirtualWorkerIds, return;)
 
-				protected :
+		// This returns the LBStrategy which should be rendered in the SpatialDebugger.
+		// Currently, this is just the default strategy.
+		virtual UAbstractLBStrategy* GetLBStrategyForVisualRendering() const
+		PURE_VIRTUAL(UAbstractLBStrategy::GetLBStrategyForVisualRendering, return nullptr;)
 
-		VirtualWorkerId LocalVirtualWorkerId;
+			protected : VirtualWorkerId LocalVirtualWorkerId;
```

Comment:
putting semi colons after the `PURE_VIRTUAL(...)` for the lines about should mean clang-format doesn't mess up the `protected:` tag here too